Dockerfile

```diff
@@ -87,7 +88,7 @@ RUN set -ex \
         /usr/share/doc-base
 
 # Setup pipeline debugging tools
-RUN pip install https://codeload.github.com/GlobalFishingWatch/pipe-tools/tar.gz/v0.2.0
+RUN pip install https://codeload.github.com/GlobalFishingWatch/pipe-tools/tar.gz/d884-2
```

Comment:
@smpiano we should wait until we merge pipe-tools and include a final tag in here.